commands/nasa_apod.py

```diff
@@ -0,0 +1,12 @@
+async def nasa_apod(ctx):
+    """ Sends a beautiful space picture of the day from NASA """
+    url = f"https://api.nasa.gov/planetary/apod?api_key={NASA_API_KEY}"
+    response = requests.get(url)
+    response = response.json()
+
+    print(response)
+
+    photo = response['url']
+    title = response['title']
+
+    await ctx.send(f"**Description:** {title}\n{photo}")
```

Comment:
Newline at end of file
```suggestion
    await ctx.send(f"**Description:** {title}\n{photo}")
    
```